ui/pages/_app.js

```diff
@@ -23,6 +23,8 @@ import { SnackbarProvider } from 'notistack';
 import PropTypes from 'prop-types';
 import React, { useCallback, useEffect } from 'react';
 import { connect, Provider } from 'react-redux';
+import { AbilityContext } from '../utils/can';
+import ability from '../utils/ability';
```

Comment:
Can you please confirm, if you missed pushing changes for adding ability file under `utils` folder?